projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -42,20 +42,17 @@ export class DetailsPageComponent implements OnInit {
       this.projectDetails = data.data
       this.submitted = data.data.status == statusType.submitted
       this.getProjectTaskStatus()
-      this.countCompletedTasks(this.projectDetails);
+      this.countCompletedTasks();
       this.calculateProgress();
       this.setActionsList()
       this.initializeTasks()
     })
   }
+  countCompletedTasks(): number {
```

Comment:
Why is this function added?

Reply:
to should the number of task completed

Reply:
Arent we doing that in calculatePregress() ?

Reply:
no its just to calculate the progress value that's it